prow/spyglass/lenses/junit/template.html

```diff
@@ -21,38 +22,127 @@
     </tr>
     <tbody id="failed-tbody">
     {{range $ix, $test := .Failed}}
+      {{$numTest := len $test.Junit}}
+      {{$firstTest := index $test.Junit 0}}
+      {{if eq $numTest 1}}
```

Comment:
Would be great to abstract some of the common HTML out of the if/else block (if possible, it looks like there's some common lines between them but I may be missing something that means they can't be factored out)

Reply:
I just checked, it didn't seem to me that there is much could be refactored out